src/scheduler/server_info.c

```diff
@@ -709,7 +709,11 @@ query_server_dyn_res(server_info *sinfo)
 				}
 
 				if (set_resource(res, buf, RF_AVAIL) == 0) {
-					return -1;
+					snprintf(buf, sizeof(buf), "Script %s returned bad output",
+							conf.dynamic_res[i].program);
+					schdlog(PBSEVENT_DEBUG, PBS_EVENTCLASS_SERVER, LOG_DEBUG,
```

Comment:
Do you want debug or error?

Reply:
We don't error out when this happens, the scheduler behaves similarly to how it does when other similarly bad things happen in the function like bad pipe for the server_dyn_res script, it moves on without erroring out. So, I think DEBUG is consistent.

Reply:
okay